service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -76,6 +76,13 @@ public class RemremGenerateController {
     @Value("${lenientValidationEnabledToUsers:false}")
     private boolean lenientValidationEnabledToUsers;
 
+    @Value("${maxEventsOfInputArray:250}")
+    private int maxSize;
```

Comment:
Rename the variable to be more expressive.

Reply:
Done